tools/msmetaenhancer/msmetaenhancer.xml

```diff
@@ -0,0 +1,53 @@
+<tool id="msmetaenhancer" name="MSMetaEnhancer" version="0.1.0+galaxy0">
+    <description>annotate MS data</description>
+    <requirements>
+        <requirement type="package" version="0.1.0">msmetaenhancer</requirement>
+    </requirements>
+
+    <command detect_errors="exit_code"><![CDATA[
+        sh ${msmetaenhancer_python_cli}
+    ]]> </command>
+
+    <configfiles>
+        <configfile name="msmetaenhancer_python_cli">
+            python3 ${__tool_directory__}/msmetaenhancer_wrapper.py \
+            --input_file "$input_file" \
+            --output_file "$output_file" \
+            --services "$services"
+        </configfile>
+    </configfiles>
+
+    <inputs>
+        <param label="Input spectra file" name="input_file" type="data" format="msp" help="Path to query spectra file in MSP format." />
+        <param name="services" type="select" label="Available services" multiple="true" display="checkboxes" optional="true">
```

Comment:
If optional is true, can you choose no service? what happens then?

Reply:
In updated version it is possible to enter no jobs, which means all possible jobs in random order will be executed.

Reply:
Is that behaviour documented?

Reply:
It is mentioned in the help section.